src/Ledger/Transaction.lagda

```diff
@@ -95,11 +95,9 @@ the transaction body are:
     ; scriptStructure = scriptStructure
     ; govParams = govParams
     }
+
   open Ledger.GovernanceActions govStructure hiding (yes; no) public
   open Ledger.Deleg             govStructure public
-
-  _ : Value → Value → Set
-  _ = _≈_
```

Comment:
This was a unit test to make sure we have not messed up the instances in all the structures above which `open` several sub-structures, although I agree it seems arbitrary (i.e. we are missing a principles way to test that instances are properly set up).

Reply:
One could argue that this is precisely what definitions named `_` signify, i.e. unit tests or examples which are kind of the same thing in the dependently-typed world.